src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-investigation-step.html

```diff
@@ -0,0 +1,201 @@
+            <p data-ng-repeat="actionDone in $ctrl.currentLineDiagnostic.getActionsDone()">
+                <span class="oui-icon oui-icon-success"
+                      aria-hidden="true"></span>
+                <span data-translate="{{:: 'tools_lineDiagnostics_investigation_step_action_' + actionDone + '_done_text' }}"></span>
+            </p>
+            <p data-ng-if="$ctrl.loading &&
+                           $ctrl.getInvestigationStepQuestions().length === 0 &&
+                           $ctrl.getInvestigationStepSpecificQuestions().length === 0">
+                <oui-spinner data-size="s"></oui-spinner>
+                <span class="ml-2" data-translate="tools_lineDiagnostics_please_wait"></span>
+            </p>
+        </div>
+
+        <!-- ACTIONS TO DO -->
+        <div data-ng-if="$ctrl.currentLineDiagnostic.data.actionsToDo &&
+                         $ctrl.currentLineDiagnostic.data.actionsToDo.length > 0 &&
+                         $ctrl.currentAction && !$ctrl.loading">
+            <label for="{{ $ctrl.currentAction }}"
+                   class="oui-label">
+                <span data-translate="{{ 'tools_lineDiagnostics_investigation_step_action_' + $ctrl.currentAction + '_title' }}"></span>
+                <oui-popover data-ng-if="$ctrl.currentAction === 'bePreparedToCheckModemSynchronization'">
+                    <button type="button" class="oui-popover-button" oui-popover-trigger></button>
+                    <oui-popover-content data-translate="tools_lineDiagnostics_detection_step_modem_question_help"></oui-popover-content>
+                </oui-popover>
+                <a class="oui-link oui-link_icon"
+                   data-ng-if="$ctrl.currentAction === 'changeProfile'"
+                   data-ui-sref="telecom.pack.xdsl" >
+                    <span data-translate="tools_lineDiagnostics_investigation_step_action_changeProfile_link"></span>
+                </a>
+                <oui-popover data-ng-if="$ctrl.currentAction === 'checkFilter'">
+                    <button type="button"
+                            class="oui-popover-button"
+                            oui-popover-trigger></button>
+                    <oui-popover-content>
+                        <p class="text-center"
+                           data-translate="tools_lineDiagnostics_investigation_step_action_checkFilter_example">
+                        </p>
+                        <img src="assets/images/diagnostic/filtre-adsl.png" />
+                    </oui-popover-content>
+                </oui-popover>
+            </label>
+            <oui-button data-variant="secondary"
+                        data-on-click="$ctrl.addActionDone($ctrl.currentAction)"
+                        data-disabled="$ctrl.loading">
+                <span data-translate="{{ 'tools_lineDiagnostics_investigation_step_action_' + $ctrl.currentAction + '_text' }}">
```

Comment:
For all this file, please use one time binding when needed (I know it's not needed here)